bounce/server/resource/club.py

```diff
@@ -137,22 +139,26 @@ class SearchClubsResponse(metaclass=ResourceMeta):
                 False,
                 'properties': {
                     'name': {
-                        'type': 'string'
+                        'type': 'string',
                     },
                     'description': {
                         'type': 'string',
                     },
                     'website_url': {
                         'type': 'string',
+                        'default': 'no link',
```

Comment:
According to the schema, all our links are optional, and we don't insert anything into those columns in the DB if they're blank, so I don't think we need defaults on links. It's better to leave them blank than to have them all default to 'no link', because that makes it easier if figure out whether to display them on front end or not (i.e. checking `if obj.website_url === null` is cleaner than checking `if obj.website_url === "no link"`).

Reply:
ok sounds good